editor/src/components/editor/actions/actions.spec.browser2.tsx

```diff
@@ -271,4 +271,155 @@ describe('actions', () => {
       })
     })
   })
+  describe('UNWRAP_GROUP_OR_VIEW', () => {
+    it(`Unwraps a group`, async () => {
```

Comment:
I think it'd be safer to rename this to "Unwraps a content-affecting element" or "Unwraps an unstyled div" because what a group is is under heavy development.

Reply:
fixed